repository/h.json

```diff
@@ -89,6 +89,18 @@
 				}
 			]
 		},
+		{
+			"name": "Hadar Theme",
+			"author": "Cristian Vásquez",
+			"details": "https://github.com/CristianVasquez13/Hadar-ST",
+			"labels": ["Themes","dark","theme","color scheme","Dark Theme","color-theme","monokai","hadar","hadar theme"],
```

Comment:
That's a bit wild of a list :). So this package is a theme, and a color scheme. You don't need to repeat the name of the package in the labels, no other packages are labelled "hadar". Also, typically, all lower case. This would make a lot more sense:

```
["theme","color scheme","dark"],
```